rust/connlib/tunnel/src/client.rs

```diff
@@ -767,6 +810,11 @@ impl RoleState for ClientState {
 
             if self.refresh_dns_timer.poll_tick(cx).is_ready() {
                 let mut connections = Vec::new();
+
+                self.peers_by_ip
+                    .iter()
+                    .for_each(|p| p.1.inner.transform.expire_dns_track());
```

Comment:
Just making sure I understand - This timer doesn't enforce the expiring, it just cleans up old DNS tracks, right? I was a little confused because the timer is 300 seconds but they're supposed to expire after only 60 seconds.

Reply:
yup! this is just to cleanup the `HashSet` containing the ids so it doesn't grow too big if a lot of dns queries never see a response.